front/components/assistant/conversation/ConversationViewerVirtuoso.tsx

```diff
@@ -153,6 +154,8 @@ const ConversationViewerVirtuoso = ({
     limit: DEFAULT_PAGE_LIMIT,
   });
 
+  const justCreated = useSearchParams().get("justCreated") === "true";
```

Comment:
I guess this is part of the big previous PR, but what do we use this queryParam for?